dump_effect_sbin.py

```diff
@@ -0,0 +1,444 @@
+import collections.abc
```

Comment:
Would this make more sense in the `tools` directory rather than root?

Reply:
Sure. Made the adjustment.